__test__/api/comment-vote.test.js

```diff
@@ -7,8 +7,8 @@ import { describeIfEndpoint } from "../helpers/conditionalTests";
 
 describeIfEndpoint(
   "PATCH",
-  "/comments/:commentId/votes/upvote",
```

Comment:
Stash your changes if you have any, and then update your forked/local repo

```bash
git stash
git add remote upstream https://github.com/microapidev/comment-microapi.git
git fetch upstream
git merge upstream/develop
```

then you can apply your stash and fix merge conflicts

```bash
git stash pop
```